app/models/program.rb

```diff
@@ -94,4 +95,15 @@ def create_difficulty_levels
                                                 color: '#EF6E69')
     true
   end
+
+  ##
+  # Check if languages string has the right format. Used as before_save.
+  #
+  def check_languages_format
+    # All white spaces are removed to allow languages to be separated by ',' and ', ' but only the first option is saved
+    if self.languages.present?
+      self.languages = self.languages.delete(' ')
+      self.languages.match(/^$|(\A[a-z][a-z](,[a-z][a-z])*\z)/).present?
```

Comment:
Does it make sense to add more than one language? I mean usually the speaker presents only in one language (sure there could be subtitles, however, I'd neglect that ...)